redisinsight/api/src/main.ts

```diff
@@ -77,7 +82,7 @@ export default async function bootstrap(apiPort?: number): Promise<IApp> {
 
   await app.listen(apiPort || port, host);
   logger.log({
-    message: `Server is running on http(s)://${host}:${port}`,
+    message: `Server is running on http(s)://${host}:${apiPort || port}`,
```

Comment:
you can use just `serverConfig.port` here. There is no need for additional checks